documentation/release-notes/topics/version_4.rst

```diff
@@ -2,6 +2,49 @@
 Version 4
 #########
 
+*******************
+Hyperview 4.2 (TBD)
+*******************
+This section covers significant changes and bug fixes in Hyperview 4.2.x since version 4.1.X
+
+.. note:: This release has changes and improvements to the Data Collector software. It is strongly recommended to update your installed Data Collectors to the latest version to maintain an optimal monitoring and discovery experience.
+
+=======================================
+Enhanced Licensed Feature: Connectivity
+=======================================
+Port management has been greatly improved:
+- The port name template is editable when adding new ports from the Information -> Network Ports
+- The port name template is editable when adding new ports from the Layout page of network devices and patch panels
+- Port names are now editable in bulk from the Information -> Network Ports Page. This will allow bulk changes to port names to align with internal or manufacturer port naming conventions
+- Ports can now be deleted in bulk from the Layout page of network devices and patch panels
+
+==================================
+Enhanced Feature: New Bulk Actions
+==================================
+Bulk actions have been added to:
+- Add network ports
+- Edit network ports
+
+======================================
+Enhanced Feature: Linux Data Collector
+======================================
+- The Linux version of the Data Collector has been improved to enhance compatibility with AES256 for SNMPv3 discovery and monitoring
+- Various internal optimizations have been added to improve performance and resource usage
+
+======================================
+Other notable changes and improvements
+======================================
+- Discovery state has been added to the Asset -> Information -> Properties page. This will allow users to tell the source of the asset and if has been discovered or manually added
```

Comment:
Missing "it" in the sentence: "This will allow users to tell the source of the asset and if has been discovered or manually added"